src/classes/HealthCheckGroupController_TEST.cls

```diff
@@ -0,0 +1,109 @@
+    * Redistributions of source code must retain the above copyright
+      notice, this List of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this List of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @group 
+* @group-content ../../ApexDocContent/
+* @description Test class for HealthCheckGroupController.
+*/
+@isTest 
+public with sharing class HealthCheckGroupController_TEST {
+    /****************************************************************************************************************************
+    * @description Test method to verify that the locateHealthCheckGroupService method returns instance of HealthCheckGroupService.
+    ****************************************************************************************************************************/
+    @isTest
+    private static void locateHealthCheckGroupServiceValid() {
+        Test.startTest();
+            HealthCheckGroupService healthChkGrouSrvInstance = HealthCheckGroupController.locateHealthCheckGroupService();
+        Test.stopTest();
+
+        System.assertEquals(HealthCheckGroupService.getInstance(), healthChkGrouSrvInstance, 'Should return the instance of HealthCheckGroupService.');
+    } 
+
+    /****************************************************************************************************************************
+    * @description Test method to verify that the getHealthCheckGroupViewModel method returns a valid HealthCheckGroupVModel.
+    ****************************************************************************************************************************/
+    @isTest 
+    private static void getHealthCheckGroupViewModelValid(){
+        HealthCheckGroupService.instance = new STUB_HealthCheckGroupServiceMixedBag();
```

Comment:
This is another case where the use of stubbing is excellent. We have a highly-variable and uncontrollable method that depends on dynamic configuration here.